save-backend/src/main/kotlin/com/saveourtool/save/backend/service/TestExecutionService.kt

```diff
@@ -224,34 +220,31 @@ class TestExecutionService(private val testExecutionRepository: TestExecutionRep
                         log.error("Test execution $testExecDto with id=$testExecutionId for execution id=$executionId cannot be updated because its status is not RUNNING")
                     })
         }
-        val lock = locks.computeIfAbsent(executionId) { Any() }
-        synchronized(lock) {
-            transactionTemplate.execute {
-                val execution = executionRepository.findById(executionId).get()
-                execution.apply {
-                    log.debug {
-                        "Updating counters in execution id=$executionId: running=$runningTests-${counters.total()}, " +
-                                "passed=$passedTests+${counters.passed}, failed=$failedTests+${counters.failed}, skipped=$skippedTests+${counters.skipped}"
-                    }
-                    runningTests -= counters.total()
-                    passedTests += counters.passed
-                    failedTests += counters.failed
-                    skippedTests += counters.skipped
+        transactionTemplate.execute {
```

Comment:
`transactionTemplate` was used here to ensure transaction is closed inside `synchronized` block. It's probably not needed since the lock is handled on the DB itself.

Reply:
removed